app/bundles/EmailBundle/Form/Type/AbTestPropertiesType.php

```diff
@@ -35,7 +32,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setOptional([
```

Comment:
`setOptional` method is deprecated. Please use `setDefined` method instead.

Reply:
I don't see it documented in https://github.com/symfony/symfony/blob/3.0/UPGRADE-3.0.md#form nor in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#

I will add it to our Gdoc